File: semestre-1/algoritmos/lista-2/MR-Alg-02-Ex-01.py
dias = int(input("Digite a quantidade de dias: "))
horas = int(input("Digite a quantidade de horas: "))
minutos = int(input("Digite a quantidade de minutos: "))
segundos = int(input("Digite a quantidade de segundos: "))

total_segundos = segundos + minutos * 60 + horas * 60 * 60 + dias * 60 * 60 * 24

print(f"Total de segundos do intervalo dado: {total_segundos}")

File: semestre-1/algoritmos/MR-Alg-02/.c/MR-Alg-02-Ex-01.c
/* Crie um programa Python que leia do usuário um valor de intervalo de
tempo expresso em número de dias, horas, minutos e segundos. O programa deve computar e
exibir a quantidade total de segundos deste intervalo de tempo informado. */

#include <stdio.h>

void main() {
    int dias, horas, minutos, segundos;
    printf("Dias: ");
    scanf("%d", &dias);

    printf("Horas: ");
    scanf("%d", &horas);

    printf("Minutos: ");
    scanf("%d", &minutos);

    printf("Segundos: ");
    scanf("%d", &segundos);

    int total_time = ((dias * 24 + horas) * 60 + minutos) * 60 + segundos;

    printf("Total de segundos: %d\n", total_time);
}